### ISSUE TITLE
List of habits should have a wrapper class

### ISSUE THREAD

**Issue by TakenouchiTR** [JACorley1/UWG-SE2-Spring22-Team1#19]
A list of habits is a common occurrence in many parts of the program.

**Comment by Steven-Carriger** [comment]
Habit-WrapperClass' PR resolves this issue once it passes and is merged into the development branch,
